misc/test-production-rules.py

```diff
@@ -1,5 +1,11 @@
 # This script is used only for CI/CD purposes
 # We assume that everything is correctly built and available in the `target` directory.
+
+# If you want to try this locally
+# 1. Build the binary locally (e.g. cargo run -r)
+# 2. Bootstrap a Python environment with requests (e.g python -mvenv venv && source venv/bin/activate && pip install requests)
+# 3. Execute the script: misc/test-production-rules.py -r <RULSET-NAME> -c target/release/datadog-static-analyzer -s target/release/datadog-static-analyzer-server 
```

Comment:
you also need to add execute permissions on the file, and remember not to commit the new permission changes. unless we are ok with making this file executable permanently

Reply:
Just execute Python directly, specified this.